lib/between_meals/knife.rb

```diff
@@ -39,6 +39,14 @@ def initialize(opts = {})
       @config = opts[:config] ||
                 "#{@home}/.chef/knife-#{@user}-taste-tester.rb"
       @knife = opts[:bin] || 'knife'
+      @knife_verb_option = ''
+      unless @logger.nil?
+        if @logger.level == Logger::DEBUG
+          @knife_verb_option = "-VV"
+        elsif @logger.level == Logger::INFO
```

Comment:
isn't it `INFO` by default? I know in grocery-delivery its `WARNING` by default, but i think in tt it's `INFO`

Reply:
In TT it's WARNING as well

Reply:
If it's fine, LMK, I will drop this pull request and publish another one with fixed linters.

Reply:
Good call. Then yes, this is perfect.

Don't drop the PR, just update it :)

Reply:
Updated it, but circle CI does not allow me to restart the tests, as I have no rights.

Reply:
it re-ran them. your lint is still failing. :)

Reply:
it'll re-run them on every push (including force-pushes)

Reply:
Fixed now. I did not know it supports force pushes.